client/blocks/importer/wordpress/upgrade-plan/index.tsx

```diff
@@ -54,7 +41,23 @@ export const UpgradePlan: React.FunctionComponent< Props > = ( props: Props ) =>
 		// If the user's email is unverified, we still want to show the trial plan option
 		migrationTrialEligibility?.error_code === 'email-unverified';
 
+	const { list: upgradePlanHostingDetailsList, isFetching: isFetchingHostingDetails } =
+		useUpgradePlanHostingDetailsList();
+
+	const plans = Plans.useSitePlans( { siteId: site.ID } );
```

Comment:
Using `usePricingMetaForGridPlans` for pricing instead of the query hook is probably better for consistency. It will also fix the issue with always showing prorated prices. There is a PR that makes it optional.

Unless there is a specific other exception, then `usePricingMetaForGridPlans` with `siteId` should give the equivalent of `useSitePlans`.

Reply:
Thanks for the suggestion, updated here: https://github.com/Automattic/wp-calypso/pull/92201/commits/e4b21382979fbc2d9898892ce5eda5ccaee6f58e

I chose `useSitePlans`, because the call looked less wordy :) `usePricingMetaForGridPlans` has other required properties that makes the call spread on several lines.